rust_dev_preview/webassembly/README.md

```diff
@@ -0,0 +1,56 @@
+# AWS SDK for Rust code examples using WebAssembly
+
+## Purpose
+
+This example demonstrates how to package in an WebAssemlby module that uses the developer preview version of the AWS SDK for Rust.
+
+## Code examples
+
+- [Show functions](./src/lib.rs) (ListFunctions)
+
+## Running the code examples
+
+### Prerequisites
+
+- You must have an AWS account, and have configured your default credentials and AWS Region as described in [https://github.com/awslabs/aws-sdk-rust](https://github.com/awslabs/aws-sdk-rust).
+
+- Install the latest stable version of [Node.js](https://nodejs.org/en/download/).
+
+- Install [wasm-pack](https://rustwasm.github.io/wasm-pack/installer/#).
+
+### ⚠ Important
+Your AWS credentials should be customized in file [www/env/credentials.js](./www/env/credentials.js) otherwise it will not make the request to the backend.
```

Comment:
You must customize your AWS credentials in the [www/env/credentials.js](./www/env/credentials.js) file. Otherwise, it will not make the request to the backend.